externs/shaka/player.js

```diff
@@ -1442,6 +1443,11 @@ shaka.extern.ManifestConfiguration;
  *   If <code>true</code>, the media source live duration
  *   set as a<code>Infinity</code>
  *   Defaults to <code> false </code>.
+ * @property {numebr} minCompletionPreloadNextUrl
+ *   Minimum completion percentage to preload the next URL. The range of
+ *   accepted values is between 0 and 100. If the value is 0 it implies that
```

Comment:
I'm not sure if completion percentage is the right metric to use for this.
90% completion percentage is going to be a lot different for an hour-long manifest, compared to a minute-long manifest. If your MPD chain includes both long and short manifests, that could be an issue.
It might make more sense to define a time value, measured in seconds till the end of the presentation.

Reply:
done!